gap/attr.gi

```diff
@@ -1582,3 +1582,125 @@ function(gr)
   od;
   return fail;
 end);
+
+InstallMethod(VertexConnectivity,
+"for a digraph",
+[IsDigraph],
+function(digraph)
+  local edges, edmondskarp, k, kappa, kappas, mindegv, Nv, outdegs, relabel,
+  sdigraph, set1v, x, y;
+
+  if DigraphNrVertices(digraph) = 0 then
+    return 0;
+  fi;
+
+  sdigraph := DigraphSymmetricClosure(DigraphRemoveAllMultipleEdges(digraph));
+  edges := DigraphEdges(sdigraph);
+  kappas := [DigraphNrVertices(digraph) - 1];
+
+  relabel := function(edge, source, sink, wvert)
+    local nedge, i;
+    nedge := [];
+
+    for i in [1, 2] do
+      if edge[i] = source then
+        nedge[i] := 1;
+      elif edge[i] = sink then
+        nedge[i] := 2;
+      else
+        nedge[i] := Position(wvert, edge[i]) * 2 + i;
+      fi;
+    od;
+    return nedge;
+  end;
+
+  kappa := function(source, sink, digraph, edges)
+    local wvert, nedges, i;
+    wvert := ShallowCopy(DigraphVertices(digraph));
+    Remove(wvert, Position(wvert, source));
```

Comment:
Replace `Position` by `PositionSorted`, since `wvert` is a sorted list, and it's much faster.